app/controllers/assignments_controller.rb

```diff
@@ -157,7 +158,7 @@ def edit
         flash_now(:notice, t('assignments.due_date.past_due_date_notice') + past_date.join(', '))
       end
     end
-
+    @assignment_duration_parts = @assignment.assignment_properties.duration_parts if @assignment.is_timed
```

Comment:
I prefer not creating another instance variable here. We setup a proxy on `assignment_properties`, so you'll be able to do `@assignment.duration_parts` where this is used in the views. Also, the only place I could see this variable being used is guarded by an `if @assignment.is_timed` already.